strato_query/base_API_query.py

```diff
@@ -734,7 +734,7 @@ def _submit_post_request(json_dict: dict,
             if retry_num == cc.MAX_RETRIES - 1:
```

Comment:
Not sure this logic is correct. For example, if I set the max to 3, then this would raise the exeception before the third retry (after the third *try*) right? Maybe just a variable rename to clarify?

Reply:
Good point, it would make 3 attempts, but only 2 retries. Thanks for catching that.